README.md

```diff
@@ -12,17 +12,30 @@ See [this get started guide](https://www.serverless.com/framework/docs/getting-s
 
 ## Usage
 
+### With [`cml comment`](https://cml.dev/doc/ref/comment)
+
+```console
+$ cml comment create --publish --publish-url={endpoint}
+```
+
 ### With `curl`
 ```console
-$ curl {endpoint} --header "Content-Type: image/png" --data-binary @image.png
+$ curl {endpoint} --data-binary @image.png --header "Content-Type: image/png"
 https://{bucket}.s3.{region}.amazonaws.com/{hash}
 ```
 
-### With [`cml comment`](https://cml.dev/doc/ref/comment)
+#### Overwriting files
+
+Storage is [content-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage) by default, naming objects after their contents' SHA-256 hash. This behavior can be overriden by means of the `Content-Seed` header, whose value will be used to calculate the mentioned hash, so the returned URL remains unchanged after overwriting the contents of the object.
 
 ```console
-$ cml comment create --publish --publish-url={endpoint} image.png
-https://{bucket}.s3.{region}.amazonaws.com/{hash}
+$ curl {endpoint} --data-binary @file_v1 --header "Content-Seed: $(uuidgen)"
+https://{bucket}.s3.{region}.amazonaws.com/8da7...2297
+$ curl {endpoint} --data-binary @file_v2 --header "$_"
+https://{bucket}.s3.{region}.amazonaws.com/8da7...2297
 ```
 
+> **Warning**
+> _Always specify a cryptographically secure `Content-Seed` value, lest allow unauthorized users to read or overwrite the uploaded files._
```

Comment:
```suggestion
> _If using `Content-Seed`, use a cryptographically secure value to prevent unauthorized users from reading/overwriting the uploaded files._
```

maybe also link to some definition of "cryptographically secure?"

Reply:
> _maybe also link to some definition of "cryptographically secure?"_

Anything that is, at least, as good as the `uuidgen` from `util-linux` in the example. Lacking a canonical definition, a healthy `head --bytes 32 /dev/random | base64` is enough.